### PR TITLE
Restructure/split file backup

### LINKED ISSUE #2: Remove all dependencies of FileBackupManager from csv_to_confluence
See requirements. The backup manager should be an own executable script.

### LINKED ISSUE #3: Update copyright version in FileBackupManager

### PR DESCRIPTION
**Pull request by LongusBongus**
i request a pull :). Also commons Klassen werden importiert und das Backup Skript beinhaltet nur FileBackupManager, ConfluenceHandler und eine Main.

### CODE REVIEW

File: src/file_backup_service.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+#      You should have received a copy of the GNU Affero General Public License
+#      along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+#
+
+
+import os
+import sys
+from abc import ABC
+from src.common import SingletonABCMeta, ConfluenceNodeMapper, ConfluenceConnection, Main
```
Comment (akomii) [inline src/file_backup_service.py]: should be

from common import ....

File: src/csv_to_confluence.py [hunk 2]
```diff
@@ -36,6 +36,7 @@
 
 from common import Main, CSVHandler, ConfluenceConnection, ConfluenceNodeMapper, ErrorCSVHandler, InfoCSVHandler, ResourceLoader, SingletonABCMeta, \
     SingletonMeta, TimestampHandler
+from src.file_backup_service import FileBackupManager
```
Comment (akomii) [inline src/csv_to_confluence.py]: This line has no use

**Review by akomii — CHANGES_REQUESTED** [review]

File: src/csv_to_confluence.py (hunk 2 above)
Comment (LongusBongus) [reply src/csv_to_confluence.py]: Class FileBackupManager is used in class ConfluencePageHandlerManager to upload a backup to confluence.

Comment (akomii) [reply src/csv_to_confluence.py]: FileBackupManager should be decoupled from csv_to_confluence.py and executed seperately

**Review by akomii — APPROVED** [review]

**Pull request merged**